storage/google/cloud/storage/bucket.py

```diff
@@ -437,6 +437,17 @@ def user_project(self):
         """
         return self._user_project
 
+    @property
+    def properties(self):
+        """Properties of the current bucket object.
+
+        Returns:
+            dict: Bucket properties transformed to dict.
+        """
+        properties = {key: self._properties[key] for key in self._changes}
+        properties["name"] = self.name
+        return properties
```

Comment:
Moved some lines from Bucket.create() to property for public accessing

Reply:
I'd prefer we didn't introduce this as a public property. It's okay for the Client class to access "private" `_properties` since it's in the same library.

Reply:
Hm, if so, we don't need this property at all, I think